scan-irb/report-en.tex

```diff
@@ -105,11 +117,21 @@
   lab} are part of a \textbf{research study} and this report describes your
   \textbf{research test result.}
 
+%- if staff_observed == "true"
 \item
-
   You collected this sample and labeled it by yourself (or for someone else) and
   returned it to the lab via a delivery service. It was not collected, labeled,
   or transported by a licensed healthcare professional like a nurse or doctor.
+
+  %- elif staff_observed == "false"
+\item
+  You collected this sample by yourself (or for someone else) and it was labeled
```

Comment:
Does this new text block correlate to the testing sweeps from this week and last week? Would those samples collected from the sweeps have staff_observed == true or false?

Reply:
Yes, this is in preparation for the kiosk (in-person) testing and the Greek sweeps. Those encounters will have staff_observed == 'True'

Writing this out made me reconsider that we may want to use python native boolean values `True`/`False` instead of lowercase "true"/"false" (although LaTeX will likely have to treat the values all as strings)

Reply:
Oh, yes you're right. I accidentally got these flipped backwards. Thank you for catching that!